infrastructre/template.yaml

```diff
@@ -52,3 +53,81 @@ Resources:
     Type: AWS::SQS::Queue
     DeletionPolicy: Retain
     UpdateReplacePolicy: Retain
+
+  ######################
+  # Querying the Record
+  ######################
+  QueryServiceRecordFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Architectures: ["x86_64"]
+      CodeUri: ../lambda/query-service-record/query-service-record.ts
+      Events:
+        TxMAInputDummyQueue:
+          Type: SQS
+          Properties: 
+            Queue: !GetAtt TxMAInputDummyQueue.Arn
+      DeadLetterQueue:
+        Type: SQS
+        TargetArn: !GetAtt QueryServiceRecordDeadLetterQueue.Arn
+      Handler: query-service-record.handler
```

Comment:
We'll need to check that the handler and file names match what we've put in the lambdas. Won't be much effort to change either way.